app/controllers/prevalidateTrader/PrevalidateTraderStartController.scala

```diff
@@ -17,24 +17,40 @@
 package controllers.prevalidateTrader
 
 import controllers.predicates.{AuthAction, AuthActionHelper, BetaAllowListAction, DataRetrievalAction}
+import models.UserAnswers
 import play.api.i18n.I18nSupport
-import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import services.PrevalidateTraderUserAnswersService
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
-import views.html.prevalidateTrader.PrevalidateTraderStartPage
+import views.html.prevalidateTrader.PrevalidateTraderStartView
 
 import javax.inject.Inject
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ExecutionContext, Future}
 
-class StartPrevalidateTraderController @Inject()(mcc: MessagesControllerComponents,
+class PrevalidateTraderStartController @Inject()(mcc: MessagesControllerComponents,
```

Comment:
Minor comment: This class could inject the navigator and do all the routing there

Reply:
Matt and I discussed this, it's a trade off between keeping all navigation in one place vs extra boilerplate (creating a new Page just for this, injecting extra things into the class, etc). What do you think? I'm happy to make the change if you think it's worth it.

Reply:
Ah yeah, I didn't think about the Page object aspect. Happy to leave as is